zhaquirks/philips/rdm002.py

```diff
@@ -69,6 +75,48 @@ class PhilipsRdm002RemoteCluster(PhilipsRemoteCluster):
     }
 
 
+class PhilipsRdm002LevelControl(CustomCluster, LevelControl):
+    """Philips RDM002 LevelControl cluster."""
+
+    def listener_event(self, method_name: str, *args) -> list[Any | None]:
+        """Blackhole requests with transition_time=8, which originate from button long-presses."""
+
+        _LOGGER.debug(
+            "%s - listener_event: method: %s - args: [%s]",
+            self.__class__.__name__,
+            method_name,
+            args,
+        )
+
+        # example args we want to mute:
+        # [(224, 6, step_with_on_off(step_mode=<StepMode.Down: 1>, step_size=255, transition_time=8))]
+        # transition_time=8 when step command is sent for long-press of button 1.
+        # transition_time=4 when dial is rotated.
+        if (
+            method_name == "cluster_command"
+            and len(args) > 2
+            and len(args[2]) > 2
+            and args[2][2] == 8
+        ):
+            _LOGGER.debug(
+                "%s::listener_event - muting level control method: %s - args: [%s]",
+                self.__class__.__name__,
+                method_name,
+                args,
+            )
+
+        else:
+            _LOGGER.debug(
+                "%s::listener_event - forwarding level control method: %s - args: [%s]",
+                self.__class__.__name__,
+                method_name,
+                args,
+            )
+            return util.ListenableMixin.listener_event(self, method_name, *args)
+
+        return []
```

Comment:
Can't we just call `super().listener_event(method_name, *args)`?

Reply:
yup, changed to use `super()`